source/changelog.txt

```diff
@@ -12,6 +12,48 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.7
+------
+
+*Released December 17, 2024*
+
+Bug Fixes:
+
+- :issue:`MONGOSH-1943` - Only ever run ``MongoshRepl.onExit`` once
+- :issue:`MONGOSH-1969` - Don't override output that came from onPrint
```

Comment:
The latter item is an internal ticket for an issue in 2.3.6 that users will never see (it's only showing in Compass's built-in REPL). I'd drop it:
```suggestion
```